documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -1910,6 +1910,33 @@ This also enables tracking DDL changes against captured tables, so if the schema
 This allows LogMiner to mine substantially faster but at the expense that DDL changes cannot be tracked.
 If the captured table(s) schema changes infrequently or never, this is the ideal choice.
 
+|[[oracle-property-log-mining-buffer-type]]<<oracle-property-log-mining-buffer-type, `+log.mining.buffer.type+`>>
+|`memory`
+|The buffer type controls how the connector will manage buffering transaction data. +
+ +
+`memory` - Uses the JVM process' heap to buffer all transaction data.
+This is the ideal choice if you don't expect a lot of long-running or large transactions. +
+The buffer state is not persisted across restarts and will need to be recreated from the current offset's SCN value. +
+ +
+`infinispan` - This uses an embedded Infinispan cache to buffer all transaction data and persist it to disk.
+This is the ideal choice if you expect long-running or large transactions.
+Since the buffer state is persisted across restarts, the buffer does not need to be recreated at restart.
+Using this mode requires the use of `log.mining.buffer.location` to designate where the cache files are to be written.
+
+|[[oracle-property-log-mining-buffer-location]]<<oracle-property-log-mining-buffer-location, `+log.mining.buffer.location+`>>
+|
+|Specifies a directory location to where the connector can read and write the buffer cache to disk.
```

Comment:
Specifies the location of the directory that the connector uses to read and write the buffer cache.